index.html

```diff
@@ -1,155 +1,165 @@
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Dienebi Ombu - Portfolio</title>
-    <link rel="stylesheet" href="styles.css" />
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap" rel="stylesheet" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
-  </head>
-  <body>
-    <header>
-      <nav class="toolbar">
-        <a class="logo" href="index.html">ebidien</a>
-        <ul class="menu-items">
-          <li><a href="#portfolio" class="menu-links">Portfolio</a></li>
-          <li><a href="#about" class="menu-links">About</a></li>
-          <li><a href="#contact" class="menu-links">Contact</a></li>
-        </ul>
-        <img class="main-menu" src="menu-button.svg" alt="Main Menu" />
-      </nav>
-    </header>
-    <section class="headline">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Dienebi Ombu - Portfolio</title>
+  <link rel="stylesheet" href="styles.css" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;600;700&display=swap"
+    rel="stylesheet" />
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" />
+</head>
+
+<body>
+  <header>
+    <nav class="toolbar">
+      <a class="logo" href="index.html">ebidien</a>
+      <ul class="menu-items">
+        <li><a href="#portfolio" class="menu-links">Portfolio</a></li>
+        <li><a href="#about" class="menu-links">About</a></li>
+        <li><a href="#contact" class="menu-links">Contact</a></li>
+      </ul>
+      <img class="main-menu" src="menu-button.svg" alt="Main Menu" />
+    </nav>
+  </header>
```

Comment:
- [ ] You have done so well so far 👍 . You implemented the `sticky` element which is an incredibly great achievement 👏 . However, you may consider fixing the `width` of the header so that it does not cause a horizontal scroll 😍 . 

- Your design 😎 
![headermy](https://user-images.githubusercontent.com/99394374/213885056-5ef13263-f190-4f69-b4ea-0273ad149c3d.PNG)

- Figma design 😎 
![header](https://user-images.githubusercontent.com/99394374/213885076-9ae6f519-b3dc-4246-862d-8829f8a8062a.PNG)